lib/Doctrine/DBAL/Logging/BacktraceLogger.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Doctrine\DBAL\Logging;
+
+class BacktraceLogger extends DebugStack
```

Comment:
I'd avoid coupling the `BacktraceLogger` to the existing `DebugStack` one, if possible

Reply:
well, the whole point here is to add the backtrace into the data stored for each query. Not coupling it would defeat the point.